### ISSUE TITLE
"Open with" should open QL with the "File System" view

### ISSUE THREAD

**Issue by elboulangero** [quodlibet/quodlibet#3543]
Hello,

It seems to me that "Open with" (from the desktop's file explorer) should open QL with the "File System" view, so that I can see the files that I just opened and that QL is now playing.

Steps to reproduce
------------------

Open your file browser (GNOME Nautilus in my case), select a few audio files, right click, and select "Open with Quod Libet".

Expected Output
---------------

- QL starts and plays the files
- QL shows me the "File System" view, and more precisely the directory where are the files I'm playing. For example, if I opened the audio file `audio.mp3` in the directory `Music`, then in the left pane the directory `Music` should be selected, and in the right pane the file `audio.mp3` should be visible.

Actual Output
-------------

- QL starts and plays the files.
- QL shows me whatever view was set the last time I used it. It could be "Paned Browser", "Album List", etc... I don't see the files that I'm playing.

Workaround: click "Browse -> File System", then hit `<Ctrl+J>` to move to the right location. Even though, it's not perfect. The file that is playing is **NOT** selected in the list of files shown in the right pane. As a result, the next track that will play will be the first of this list: it's actually the track that is currently playing! It is played twice!

Test System
-----------

- QL `4.3.0`
- Debian Sid
- GNOME desktop

Additional notes
-----------

If the behavior I expect make sense, I can take a bit of time to investigate and maybe provide a PR. Thx!

**Comment by declension** [comment]
Note you can _partially_ work around this: `quodlibet --run --set-browser FileSystem`

**Comment by elboulangero** [comment]
Indeed, it's better, then I just have to type `<Ctrl+J>` when QL shows up. Thx for the hint!

**Comment by Ironbalden** [comment]
@elboulangero I don't think this is a good idea, changing the browser can be confusing to the end user. 
It doesn't solve the problem either, since we are just switching from playing a single file to playing the whole folder, what I expect is that Quodlibet plays only the songs I have chosen.
I think it would be better to add the files to the queue, it is less intrusive and has the correct output of playing only the songs selected.

**Comment by elboulangero** [comment]
@Ironbalden Agreed, what I suggest does not really solve the problem, I realized that it's a bit more complicated than I thought. Using the queue as you suggest seems better.

Maybe another issue on the way is that at the moment, Quodlibet is started as such:

    /usr/share/applications/io.github.quodlibet.QuodLibet.desktop
    ----
    Exec=quodlibet --run --play-file %U

So when I select multiple files and play it with Quodlibet, I'm not even sure that QL considers all the files. The help seems to suggest that it only consider the first file:

    --play-file=filename                     Play a file

**Comment by Ironbalden** [comment]
we have the command line input `--enqueue-files` that may come in handy but I had no luck to make it work with %U or %F.
